docker/Dockerfile

```diff
@@ -1,17 +1,15 @@
-FROM ubuntu:jammy@sha256:f9d633ff6640178c2d0525017174a688e2c1aef28f0a0130b26bd5554491f0da AS TMP
+FROM --platform=${TARGETPLATFORM} ubuntu:jammy-20240212 AS builder
```

Comment:
Is there a reason for removing the SHA pinning here?

Reply:
In order for buildx to fetch the proper image each time for the respective arch build